README.md

```diff
@@ -605,6 +605,16 @@ import CookieStore from 'ember-simple-auth/session-stores/cookie';
 export default CookieStore.extend();
 ```
 
+If you are using the [`OAuth2PasswordGrantAuthenticator`](http://ember-simple-auth.com/api/classes/OAuth2PasswordGrantAuthenticator.html), you must add `node-fetch` to your list of FastBoot whitelisted dependencies in `package.json`:
```

Comment:
Can you format the text so that it adheres to the 80 chars line limit for consistency with the rest of the README?